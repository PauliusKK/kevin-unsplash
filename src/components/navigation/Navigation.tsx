import React from 'react';
import Logo from '../../assets/images/logo.svg';
import PhotosIcon from '../../assets/images/photos2.svg';
import FavouritesIcon from '../../assets/images/favourites2.svg';

import './scss/navigation.scss';

export const Navigation = () => (
  <div className="navigation">
    <div className="logo">
      <img src={Logo} alt="kevin." />
    </div>
    <div className="menu">
      <div className="button">
        <img src={PhotosIcon} />
      </div>
      <div className="button">
        <img src={FavouritesIcon} />
      </div>
    </div>
  </div>
);