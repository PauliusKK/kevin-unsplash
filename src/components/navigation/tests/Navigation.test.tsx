import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import { Navigation } from '../Navigation';

describe('ComponentNavigaton', () => {
  test('renders correctly', () => {
    const { asFragment } = render(
      <Router>
        <Navigation />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});