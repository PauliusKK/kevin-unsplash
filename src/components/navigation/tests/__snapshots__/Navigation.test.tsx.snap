// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentNavigaton renders correctly 1`] = `
<DocumentFragment>
  <div
    class="navigation"
  >
    <a
      class="navigation__logo"
      href="/"
    >
      <img
        alt="kevin."
        src="logo.svg"
      />
    </a>
    <div
      class="navigation__menu"
    >
      <a
        class="navigation__button"
        href="/photos"
      >
        <img
          alt="photos"
          src="photos.svg"
        />
      </a>
      <a
        class="navigation__button"
        href="/liked"
      >
        <img
          alt="liked"
          src="heart.svg"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
